### PR TITLE
[Goonchem] FirstAid Transitions and Chem Adjustsments+Fixes

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Lots of Goon chemistry related stuff

Transition the first aid kits (minus the advanced ones) from StarTrek chem to Goonchem

- First aid kits generally contain goonchems instead of startrek chems
 - Adds new first aid kit that's for treating brute damage
 - Advanced First Aid kits will remain, for now, as doctor spawns

- Auto-injectors now contain epinephrine instead of inaprovaline

Other transition stuffs
- Nuke Ops pills are not Initro pills as opposed to cyanide, as cyanide doesn't have the "umph" it used to
- Medvend now has insulin syringes
- Sulfuric acid, as a reagent, removed from the dispenser
 - Adds in sulfuric acid recipe to make sulfuric instead
- space cleaner recipe updated to prevent conflicts.

Other Chem Changes
- Silver sulfa doesn't make you scream anymore (not supposed to)
- Corrects Salicylic acid's misspelling
- updates the overloaded shell's recipe to what it's meant to be (black powder as opposed to potassium+water)
- Updates sugar so it behaves accurately to Goon standards
 - having over 200 units in you will induce a diabetic coma, it has a small chance to decay into epinephrine, and it has a chance to reduce stuns/paralysis/weakens; it also makes you super jittery if you have it in you for a long time
- Adjusts corn syrup's numbers to be more accurate.
- Adds in Very High Fructose Corn syrup
 - Basically corn syrup, only stronger

Nonchem Changes
- Updates the gauze icon to look more like gauze as opposed to a bruise pack
 - Players are still associating the bruise pack with something that heals as opposed to stops bleeding; hopefully this will help reinforce that it's for wound bandaging only.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -752,9 +752,21 @@ datum
...
 				M.nutrition += 1*REM
+				if(prob(4))
+					M.reagents.add_reagent("epinephrine", 1.2)
+				if(prob(50))
+					M.AdjustParalysis(-1)
+					M.AdjustStunned(-1)
+					M.AdjustWeakened(-1)
+				if(current_cycle >= 90)
+					M.jitteriness += 10
+				if(volume > overdose_threshold)
```
Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents.dm]: I'm converting this to the dedicated overdosing proc in a few.
